tests/cache/models/samples/get.js

```diff
@@ -26,640 +27,776 @@ const enableRedisSampleStore =
-    api.get(path)
-    .set('Authorization', token)
-    .expect(constants.httpStatus.OK)
-    .end((err, res) => {
-      if (err) {
-        return done(err);
-      }
-
-      for (let i = res.body.length - 1; i >= 0; i--) {
-        const { updatedAt, createdAt } = res.body[i];
-        expect(createdAt).to.equal(new Date(createdAt).toISOString());
-        expect(updatedAt).to.equal(new Date(updatedAt).toISOString());
-      }
-
-      done();
+  describe('with returnUser toggle on, should return user object with ' +
+    'profile field: ', () => {
+    const Aspect = tu.db.Aspect;
+    const Subject = tu.db.Subject;
+    const Sample = tu.db.Sample;
+    let a1;
+    let a2;
+    let s1;
+    let s2;
+    let s3;
+    before((done) => {
+      tu.toggleOverride('returnUser', true);
+      Aspect.create({
+        isPublished: true,
+        name: `${tu.namePrefix}Aspect1`,
+        timeout: '30s',
+        valueType: 'NUMERIC',
+        criticalRange: [0, 1],
+        relatedLinks: [
+          { name: 'Google', value: 'http://www.google.com' },
+          { name: 'Yahoo', value: 'http://www.yahoo.com' },
+        ],
+      })
+      .then((created) => (a1 = created))
+      .then(() => Aspect.create({
+        isPublished: true,
+        name: `${tu.namePrefix}Aspect2`,
+        timeout: '10m',
+        valueType: 'BOOLEAN',
+        okRange: [10, 100],
+      }))
+      .then((created) => (a2 = created))
+      .then(() => Subject.create({
+        isPublished: true,
+        name: `${tu.namePrefix}Subject1`,
+      }))
+      .then((created) => (s1 = created))
+      .then(() => Subject.create({
+        isPublished: true,
+        name: `${tu.namePrefix}Subject2`,
+        parentId: s1.id,
+      }))
+      .then((created) => (s2 = created))
+      .then(() => Subject.create({
+        isPublished: true,
+        name: `${tu.namePrefix}Subject3`,
+        parentId: s1.id,
+      }))
+      .then((created) => (s3 = created))
+      .then(() => Sample.create({
+        provider: userId,
+        messageCode: '25',
+        subjectId: s2.id,
+        aspectId: a1.id,
+        value: '0',
+        relatedLinks: [
+          { name: 'Salesforce', value: 'http://www.salesforce.com' },
+        ],
+      }))
+      .then(() => Sample.create({
+        provider: userId,
+        messageCode: '50',
+        subjectId: s2.id,
+        aspectId: a2.id,
+        value: '50',
+        relatedLinks: [
+          { name: 'Salesforce', value: 'http://www.salesforce.com' },
+        ],
+      }))
+      .then(() => Sample.create({
+        provider: userId,
+        previousStatus: 'Critical',
+        messageCode: '10',
+        subjectId: s3.id,
+        aspectId: a1.id,
+        value: '5',
+        relatedLinks: [
+          { name: 'Salesforce', value: 'http://www.salesforce.com' },
+        ],
+      }))
+      .then(() => samstoinit.populate())
+      .then(() => done())
+      .catch(done);
     });
```

Comment:
the setup of samples takes up a lot of space. The setup is identical to rtu.popoulateRedis, except for the `provider: userId`